### PR TITLE
[IMP] improvements chart of account of china

### PR DESCRIPTION
**Pull request by jeffery9**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_cn/data/account.account.template.csv [hunk 1]
```diff
@@ -1,77 +1,184 @@
...
-"l10n_cn_6403","Operating Taxes and Surcharges","6403","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6601","Selling Expenses","6601","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6602","Management Expenses","6602","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6603","Financial Expenses","6603","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6701","Assets impairment Loss","6701","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6711","Non-operating expenses","6711","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6801","Income Tax Expense","6801","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6901","Prior year income adjustment","6901","expense","l10n_cn.l10n_chart_china_small_business","False"
+"id","name","code","l10n_cn_type","chart_template_id/id","reconcile",parent_id/id
+"l10n_cn_1001","库存现金","1001","asset_cash","l10n_cn.l10n_chart_china_small_business","False",
```
Comment (vin-odoo) [inline addons/l10n_cn/data/account.account.template.csv:2]: We avoid using localized terms in our data files like it was done before.
The standard way of handling it is to keep the names in English in the files and provide the translations in the translation files.

File: addons/l10n_cn/data/account.group.template.csv [hunk 2]
```diff
@@ -0,0 +1 @@
+id,code_prefix_start,code_prefix_end,name,chart_template_id/id
```
Comment (vin-odoo) [inline addons/l10n_cn/data/account.group.template.csv:1]: This file is unused. Also please make sure that all files end with a newline :)

**Review by vin-odoo — COMMENTED** [review]
Hi, I forwarded this to our Product Owner TSB, who will take a functional look into it before I do a proper review.

File: addons/l10n_cn/data/account.group.template.csv (hunk 2 above)
Comment (jeffery9) [reply addons/l10n_cn/data/account.group.template.csv]: have add group templates

File: addons/l10n_cn/data/account.account.template.csv (hunk 1 above)
Comment (jeffery9) [reply addons/l10n_cn/data/account.account.template.csv:2]: localized  should be better understood for developer

Comment (vin-odoo) [reply addons/l10n_cn/data/account.account.template.csv:2]: If the developer can read it :)
Odoo is maintained by people from around the world, and most people can read and write in English which is why it is the language that should be used for code/comments/data by default.

**Comment by jeffery9** [comment]
There are a lots of terms need to translated into English, but those terms in English never use by accountants.


--------------------------------------------------
jeffery 陈帆
手机 13913269807
微信 同手机号

2022年12月5日 15:48，Nicolas Viseur ***@***.******@***.***>> 写道：



@vin-odoo commented on this pull request.

________________________________

In addons/l10n_cn/data/account.account.template.csv<https://github.com/odoo/odoo/pull/107022#discussion_r1039256873>:

> -"l10n_cn_6051","Other Business Income","6051","income_other","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6101","Gains and Losses of fair value change","6101","income_other","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6111","Income from investment","6111","income_other","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6301","Non-operating Income","6301","income_other","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6401","Main Business Cost","6401","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6402","Other Operating Costs","6402","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6403","Operating Taxes and Surcharges","6403","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6601","Selling Expenses","6601","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6602","Management Expenses","6602","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6603","Financial Expenses","6603","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6701","Assets impairment Loss","6701","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6711","Non-operating expenses","6711","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6801","Income Tax Expense","6801","expense","l10n_cn.l10n_chart_china_small_business","False"
-"l10n_cn_6901","Prior year income adjustment","6901","expense","l10n_cn.l10n_chart_china_small_business","False"
+"id","name","code","l10n_cn_type","chart_template_id/id","reconcile",parent_id/id
+"l10n_cn_1001","库存现金","1001","asset_cash","l10n_cn.l10n_chart_china_small_business","False",


If the developer can read it :)
Odoo is maintained by people from around the world, and most people can read and write in English which is why it is the language that should be used for code/comments/data by default.

**Comment by rugo-odoo** [comment]
Sorry but, as @vin-odoo explained, you **have** to translate your files if you want this to be merged.

**Pull request closed without merging**